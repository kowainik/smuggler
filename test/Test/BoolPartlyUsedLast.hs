module Test.BoolPartlyUsedLast where

import Data.Bool (bool, not)

foo = not
