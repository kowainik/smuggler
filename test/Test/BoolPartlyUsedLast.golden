module Test.BoolPartlyUsedLast where

import Data.Bool ( not)

foo = not
